src/test/scala/com/amazon/deequ/profiles/ColumnProfilerTest.scala

```diff
@@ -400,4 +400,55 @@ class ColumnProfilerTest extends WordSpec with Matchers with SparkContextSpec
     }
   }
 
+  "return correct profile for the Titanic dataset" in withSparkSession { session =>
+    val data = session.read.format("csv")
+      .option("inferSchema", "true")
+      .option("header", "true")
+      .load("test-data/titanic.csv")
+
+    val columnProfiles = ColumnProfiler.profile(data)
+
+    val expectedProfiles = List(
+      StandardColumnProfile(
+        "PassengerId",
+        1.0,
+        891,
+        DataTypeInstances.Integral,
+        false,
+        Map(),
```

Comment:
Please use Map.empty instead of Map() here and in the subsequent calls

Reply:
ok